wrappers/Accordion_object.js

```diff
@@ -0,0 +1,67 @@
+var Wrapper = require('./Wrapper');
+
+/**
+ * Representing browser Accordion object
+ * @class
+ */
+class Accordion_object extends Wrapper{
+  constructor(selector) {
+    super(selector);
+    this.selector = selector;
+  }
+
+/**
+ * expanding section.
+ * function will expand section by clicking on accordion icon.
+ * @function
+ */
+  expand() {
+    console.log(`>>> Trying to expand section`);
+    let accordionIcon = $(this.selector).$('i').getAttribute('class');
+    console.log(accordionIcon);
+    if (accordionIcon.includes('fa-chevron-up')) {
+      console.log('>>> Section was already expanded');
+      return
+    }
+    let i = 1;
+    while (accordionIcon.includes('fa-chevron-down')){
```

Comment:
we can put multiple conditions in while loop
like in https://stackoverflow.com/questions/35274287/while-loop-multiple-conditions-dont-check-both-conditions?rq=1